pinging @apaszke about the failing JIT test:

It seems that the arg order of an Add is swapped. Should I just fix the test? Or are there more things I need to worry about? 

The error is:
```
20:06:26 OK
20:06:26 Running JIT tests
20:06:27 s...s..s...ss.s..s.....sx...x...F.s...........s...s.s..........
20:06:27 ======================================================================
20:06:27 FAIL: test_input_pruning (_main_.TestJit)
20:06:27 Check that stage 1 will return only one value
20:06:27 ----------------------------------------------------------------------
20:06:27 Traceback (most recent call last):
20:06:27   File "test_jit.py", line 1080, in test_input_pruning
20:06:27     self.assertExpected(str(fn.graph_for(x, y)))
20:06:27   File "/var/lib/jenkins/workspace/test/common.py", line 376, in assertExpected
20:06:27     self.assertMultiLineEqual(expected, s)
20:06:27 AssertionError: 'grap[304 chars]le(5, 5) = add[alpha={1}](%6, %5)\n  return (%2, %3, %7);\n}\n' != 'grap[304 chars]le(5, 5) = add[alpha={1}](%5, %6)\n  return (%2, %3, %7);\n}\n'
20:06:27   graph(%0 : Double(5, 5)
20:06:27         %1 : Double(5, 5)
20:06:27         -------- stage 1 --------
20:06:27         %4 : Double(5, 5)
20:06:27         %5 : Double(5, 5)) {
20:06:27     %2 : Double(5, 5) = mul(%0, %1)
20:06:27     %3 : Double(5, 5) = add[alpha={1}](%0, %1)
20:06:27     ---------------- stage 1 ----------------
20:06:27     %6 : Double(5, 5) = mul(%4, %1)
20:06:27 -   %7 : Double(5, 5) = add[alpha={1}](%6, %5)
20:06:27 ?                                        ----
20:06:27 +   %7 : Double(5, 5) = add[alpha={1}](%5, %6)
20:06:27 ?                                       ++++
20:06:27     return (%2, %3, %7);
20:06:27   }
20:06:27
```

--- a/test/expect/TestJit.test_input_pruning.expect
+++ b/test/expect/TestJit.test_input_pruning.expect
@@ -7,6 +7,6 @@ graph(%0 : Double(5, 5)
   %3 : Double(5, 5) = add[alpha={1}](%0, %1)
   ---------------- stage 1 ----------------
   %6 : Double(5, 5) = mul(%4, %1)
-  %7 : Double(5, 5) = add[alpha={1}](%6, %5)
+  %7 : Double(5, 5) = add[alpha={1}](%5, %6)
   return (%2, %3, %7);
 }

